File: app/src/main/java/id/ac/polinema/seameo/ecanteen/contract/ListResultContract.java
/*
 * Copyright (c) 2019.
 *
 * Author: Mahatta Maulana
 * Github: https://github.com/hattamaulana
 *
 * Last Modified at 6/21/19 10:45 AM
 */

package id.ac.polinema.seameo.ecanteen.contract;

import id.ac.polinema.seameo.ecanteen.presenter.BasePresenter;
import id.ac.polinema.seameo.ecanteen.view.BaseView;

public class ListResultContract {
    public interface Presenter extends BasePresenter {
        void update(String id, String data);
    }

    public interface View extends BaseView { }
}

File: app/src/main/java/id/ac/polinema/seameo/ecanteen/contract/ListResultContract.kt
/*
 * Copyright (c) 2019.
 *
 * Author: Mahatta Maulana
 * Github: https://github.com/hattamaulana
 *
 * Last Modified at 6/21/19 10:45 AM
 */

package id.ac.polinema.seameo.ecanteen.contract

import id.ac.polinema.seameo.ecanteen.presenter.BasePresenter
import id.ac.polinema.seameo.ecanteen.view.BaseView

class ListResultContract {
    interface Presenter : BasePresenter {
        fun update(id: String, data: String)
    }

    interface View : BaseView
}
